### PR TITLE
[feat](fe) Add `ignore_bdbje_log_checksum_read` for BDBEnvironment

### PR DESCRIPTION
**Pull request by swjtu-zhanglei**
* https://forums.oracle.com/ords/apexds/post/je-log-checksumexception-2812

* When meeting disk damage or other reason described in the oracle forums and fe cannot start due to `com.sleepycat.je.log.ChecksumException`, we add a param `ignore_bdbje_log_checksum_read` to ignore the exception, but there is no guarantee of correctness for bdbje kv data

## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by SWJTU-ZhangLei** [comment]
run buildall

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -2488,6 +2488,9 @@ public class Config extends ConfigBase {
             options = {"default", "ranger-doris"})
     public static String access_controller_type = "default";
 
+    // https://forums.oracle.com/ords/apexds/post/je-log-checksumexception-2812
+    @ConfField
+    public static boolean ignore_bdbje_log_checksum_read = false;
```
Comment (gavinchou) [inline fe/fe-common/src/main/java/org/apache/doris/common/Config.java:2493]: Add a comment here to describe when to set it to true, and what's the defects.

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Review by gavinchou — APPROVED** [review]

**Comment by dataroaring** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**